client/src/components/devices/ChooseAuthentication.vue

```diff
@@ -43,22 +43,25 @@
         >
           <authentication-card
             @click="onMethodSelected(DeviceSaveStrategyTag.Password)"
-            :state="AuthenticationCardState.Default"
+            :state="getAuthCardState(AvailableDeviceTypeTag.Password)"
             :auth-method="DeviceSaveStrategyTag.Password"
           />
         </ion-radio>
 
         <!-- TO DO: uncomment for Smartcard -->
         <!-- <ion-radio
-          v-if="false"
           class="item-radio radio-list-item"
           label-placement="end"
           justify="start"
+          :value="DeviceSaveStrategyTag.Smartcard"
```

Comment:
Instead of commenting, could you add a const variable in the component, like `const SMARTCARD_ENABLED = false` and use a v-if? This makes it so that the editor and the linter can recognize that piece of code.